src/shared/Config/Config.cpp

```diff
@@ -18,19 +18,24 @@
 
 #include "Config.h"
 #include "Policies/Singleton.h"
+#include "Util/Util.h"
 #include <mutex>
 
 #include <boost/algorithm/string.hpp>
 
 #include <unordered_map>
 #include <string>
 #include <fstream>
+#include <cstdlib>
 
 INSTANTIATE_SINGLETON_1(Config);
 
-bool Config::SetSource(const std::string& file)
+std::optional<std::string> EnvVarForIniKey(std::string const&, std::string const&);
```

Comment:
This will require an include for ```<optional>``` that's currently missing here. I'm sure it gets included indirectly somehow because the PR builds, but let's not rely on that.